docs/source/actor_visualization.rst

```diff
@@ -34,6 +37,12 @@ Available Settings
     If `False`, the server stops after the last MUSCLE message is received. Defaults to `False`.
   - ***open_browser***: (bool) If `True`, automatically opens a new tab in your web 
     browser to the visualization page upon starting. Defaults to `True`.
+  - ***automatic_mode***: (bool) If `True`, time-dependent IDS quantities can automatically
+    be visualized using a dropdown menu. Data will automatically be extracted, and an
+    appropriate plot for the quantity will be selected.
+  - ***automatic_extract_all***: (bool) This only works if automatic_mode is enabled. 
+    If `True`, automatically extracts all time dependent quantities of the IDS. 
+    If `False`, the data will only start to be extracted as soon as a new plot is selected. 
```

Comment:
Maybe warn about performance? This might read to some people like you should just always set it to true